parts/2-ngi_pipeline/slides.md

```diff
@@ -17,17 +39,26 @@ it arrived to the analysis cluster and has been long-term archived before removi
 In the early times (this is about 3 years ago...) this whole process was done manually,
 this is nowadays impossible.
 
-_NOTE: A slide with a flowchart for TACA will go with this explanation_
 
 **TACA - Tool for the Automated Cleanup and Analysis** is a tool completely written
-in Python 2.7 that does precisely this. When a machine has finished sequencing, TACA
+in Python 2.7 that does precisely this among other stuff. When a machine has finished sequencing, TACA
```

Comment:
If you're going to _say_ this I think it just adds confusion - no one _really_ cares so cut corners and simplify :smile: